index.html

```diff
@@ -88,49 +94,241 @@
+  function make_x_axis() {        
+    return d3.svg.axis()
+        .scale(x)
+         .orient("bottom")
+         .ticks(5)
+  }
+  function make_y_axis() {        
+    return d3.svg.axis()
+        .scale(y)
+        .orient("left")
+        .ticks(5)
+  }
   // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+  
+  // EVENT HANDLERS
+    d3.select('#year-range').on('change', function () {
+      options.yearRange = d3.event.target.value;
+      update();
+    });
 
   function setup() {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.employment; }))
+      .range([height, 0])
+      .nice();
+    
     // AXES
-
-    // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
-      update();
-    });
+    //Define X axis
+      xAxis = d3.svg.axis()
+                .scale(x)
+                .orient("bottom")
+                .ticks(5);
+    //Define Y axis
+      yAxis = d3.svg.axis()
+                .scale(y)
+                .orient("left")
+                .ticks(5);
+    
+   // Creating (printing the Axxis)
+    //Create X axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(0," + (height - xpadding) + ")")
+        .call(xAxis)
+        .append("text")
+         .text("employment (%)")
+         .attr("x", width - 63)
+         .attr("y", 0)
+         .attr("font-family", "sans-serif")
+         .attr("font-size", "11px")
+         .attr("fill", "red");
+      
+      //Create Y axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(" + ypadding + ",0)")
+        .call(yAxis)
+        .append("text")
+          .text("unemployment (%)")
+          .attr("x", 0)
+          .attr("y", 0)
+          .attr("font-family", "sans-serif")
+          .attr("font-size", "11px")
+          .attr("fill", "blue");
+           
+      
+    //CALLING THE GRID
+    svg.append("g")         
+        .attr("class", "grid")
+        .attr("transform", "translate(0," + height + ")")
+        .call(make_x_axis()
+            .tickSize(-height, 0, 0)
+            .tickFormat("")
+        )
+    svg.append("g")         
+        .attr("class", "grid")
+        .call(make_y_axis()
+            .tickSize(-width, 0, 0)
+            .tickFormat("")
+        )
   }
-
+  
   function update() {
```

Comment:
This opening curly brace is never closed, which prevents the script from running.